### PR TITLE
libfm, bump revision after changes to util-macros and xorg-proto

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libfm/libfm-1.3.0.2.recipe [hunk 1]
```diff
@@ -12,16 +12,24 @@ COPYRIGHT="
 	Andriy Grytsenko (LStranger)
 	"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/lxde/libfm/archive/$portVersion.tar.gz"
```
Comment (alaviss) [inline x11-libs/libfm/libfm-1.3.0.2.recipe]: We could switch this package to use the release source instead

    https://downloads.sourceforge.net/pcmanfm/libfm-$portVersion.tar.xz

It contains a pre-generated `configure` script, so we can get rid of autoconf and friends

**Comment by Begasus** [comment]
menu-cache requires libfm-extra, libfm requires menu-cache (as it is at the moment)? ...

**Comment by Begasus** [comment]
pushed new commit to the PR

File: x11-libs/libfm/libfm-1.3.0.2.recipe (hunk 1 above)
Comment (Begasus) [reply x11-libs/libfm/libfm-1.3.0.2.recipe]: download from sf ends up in a double check: CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /sources/libfm-1.3.0.2/missing autoconf

**Comment by scottmc** [comment]
@Begasus can you review this PR versus the latest recipe and see what may need to be kept from each?  This PR is reporting conflicts now.

**Comment by Begasus** [comment]
@scottmc I'll look it over, my guess is the changes to intltoolize done recently

**Comment by Begasus** [comment]
@korli could you check this, I think it should be ok, no functional changes (added libVersion(Compat) and defineDeBugInfoPackage)?

**Comment by Begasus** [comment]
@korli could this be merged (or closed)?

**Pull request merged**